projects/packages/videopress/src/client/admin/components/video-quick-actions/style.module.scss

```diff
@@ -0,0 +1,18 @@
+.actions {
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	gap: calc( var(--spacing-base) * 2 ); // 18px
```

Comment:
This value is not correct. 

<img width="391" alt="image" src="https://user-images.githubusercontent.com/77539/188190905-5c1243a2-62d6-4093-bdca-293fc4317506.png">

It should be `column-gap: var( --spacing-base );`.

From Figma file:

List | Grid
------|------
<img width="321" alt="image" src="https://user-images.githubusercontent.com/77539/188191406-9e105f5f-19b9-4b69-98ed-63f5ca24ee7a.png"> | <img width="325" alt="image" src="https://user-images.githubusercontent.com/77539/188191479-93389d2e-41ea-4a18-b32c-67fe42bf7c7b.png">

Reply:
Thanks for catching it! Plus this point, the size was wrong too, it should be `small`, and not `normal`, since the icon should have `24x24`.

`Figma`

![image](https://user-images.githubusercontent.com/1663717/188195340-47fa20ee-aa7b-406e-80f1-49867e45840b.png)

Fixed it, it now uses `small`, which reduced the icon size to `24x24`.

And applies the gap of `8px`, with that we have `16px` between each element, `4px` (of each padding) + `8px` (of gap)